pkg/ipamfetcher/custom_field.go

```diff
@@ -0,0 +1,23 @@
+package ipamfetcher
+
+type customField struct {
+	Key   string      `json:"key"`
+	Value interface{} `json:"value"`
+	Notes string      `json:"notes"`
+}
+
+type customFields []customField
+
+// GetValue retrieves a string value from a set of custom fields
+func (c customFields) GetValue(key string) string {
+	for _, field := range c {
+		if field.Key == key {
+			loc, ok := field.Value.(string)
+			if !ok {
```

Comment:
question: In what cases would the value not be a string, and why do we not care about those values?

Reply:
currently we're only interested in string values: Location and Description specifically, these are just arbitrary values from what I can tell